io.mpv.Mpv.yml

```diff
@@ -0,0 +1,165 @@
+---
+app-id: io.mpv.Mpv
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: mpv
+rename-desktop-file: mpv.desktop
+rename-icon: mpv
+
+
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--device=all"
+- "--share=network"
+- "--socket=pulseaudio"
+- "--filesystem=host:ro"
+- "--env=LC_NUMERIC=C"
+- "--env=XDG_DATA_DIRS=/usr/share:/app/share/"
```

Comment:
This is wrong. You shouldn't override `XDG_DATA_DIRS`, Flatpak is already setting this correctly. It's possible that you're breaking here fontconfig caching of host-installed fonts, breaking subtitles rendering.

Reply:
> This is wrong. You shouldn't override `XDG_DATA_DIRS`, Flatpak is already setting this correctly. It's possible that you're breaking here fontconfig caching of host-installed fonts, breaking subtitles rendering.

https://github.com/fastrizwaan/flathub/commit/37e5f80538f5dd88579b6101075077a332403b09